web/src/assets/styles/blocks.scss

```diff
@@ -743,6 +743,51 @@ section [data-type="agama/reminder"] {
   }
 }
 
+[data-type="agama/field"] {
+  > div:first-child {
+    font-size: var(--fs-large);
+
+    button {
+      padding-inline: 0;
+    }
+
+    button:hover {
+      color: var(--color-link-hover);
+      fill: var(--color-link-hover);
+    }
+
+    button b, button:hover b {
+      text-decoration: underline;
+      text-underline-offset: var(--spacer-smaller);
+    }
+  }
+
+  > div:nth-child(n+2) {
+    margin-inline-start: calc(var(--icon-size-s) + 1ch);
+  }
+
+  > div:nth-child(2) {
+    color: gray;
+    font-size: var(--fs-medium);
+  }
+
+  > div:nth-child(n+3) {
+    margin-block-start: var(--spacer-small);
+  }
+
+  &.on {
+    button {
+      fill: var(--color-link-hover);
+    }
+  }
+
+  hr {
+    margin-block: var(--spacer-normal);
+    border: 0;
+    border-bottom: thin dashed var(--color-gray);
+  }
+}
+
```

Comment:
Let's start with this. Later we can try to go for a better solution with grids and subgrids or other _advanced_ CSS stuff.